testsuite/MDAnalysisTests/topology/test_pdb.py

```diff
@@ -244,3 +247,104 @@ def test_PDB_metals():
+ATOM      8  ND2 ASN A   1     -11.806   3.406  -3.543  1.00  0.00           N
+ATOM      9  H1  ASN A   1      -8.330   3.957   0.261  1.00  0.00           H
+ATOM     10  H2  ASN A   1      -8.740   5.068  -0.889  1.00  0.00           H
+ATOM     11  H3  ASN A   1      -9.877   4.041  -0.293  1.00  0.00           H
+ATOM     12  HA  ASN A   1      -8.930   2.162  -1.239  1.00  0.00           H
+ATOM     13  HB2 ASN A   1      -9.310   4.417  -3.193  1.00  0.00           H
+ATOM     14  HB3 ASN A   1      -9.108   2.719  -3.679  1.00  0.00           H
+ATOM     15 HD21 ASN A   1     -11.572   3.791  -4.444  1.00  0.00           H
+ATOM     16 HD22 ASN A   1     -12.757   3.183  -3.294  1.00  0.00           H
+TER      17
+HETATM   18 CU    CU A   2      03.000  00.000  00.000  1.00 00.00          CU
+HETATM   19 FE    FE A   3      00.000  03.000  00.000  1.00 00.00          Fe
+HETATM   20 Mg    Mg A   4      03.000  03.000  03.000  1.00 00.00          Mg
+HETATM   21 Ca    Ca A   5      00.000  00.000  03.000  1.00 00.00          CA
+TER      22
+HETATM 1609  S   DMS A 101      19.762  39.489  18.350  1.00 25.99           S
+HETATM 1610  O   DMS A 101      19.279  37.904  18.777  1.00 23.69           Ox
+HETATM 1611  C1  DMS A 101      21.344  39.260  17.532  1.00 24.07           C
+HETATM 1612  C2  DMS A 101      18.750  40.066  17.029  1.00 20.50           C
+HETATM 1613  S   DMS A 102      22.157  39.211  12.217  1.00 27.26           S1
+HETATM 1614  O   DMS A 102      20.622  38.811  11.702  1.00 25.51           O
+HETATM 1615  C1  DMS A 102      22.715  40.764  11.522  1.00 26.00           C
+HETATM 1616  C2  DMS A 102      22.343  39.515  13.971  1.00 25.46           C
+TER    1617
+"""
+
+def test_PDB_elements():
+    """The elements attribute is equipped to obtain the element symbols,
+        if the PDB file contains a valid element symbol
+    """
+    u = mda.Universe(StringIO(PDB_elements), format='PDB')
+    element_list = np.array(['N', 'C', 'C', 'O', 'C', 'C', 'O', 'N', 'H',
+                             'H', 'H', 'H', 'H', 'H', 'H', 'H', 'Cu', 'Fe',
+                             'Mg', 'Ca', 'S', 'O', 'C', 'C', 'S', 'O', 'C',
+                             'C'], dtype=object)
+    assert_equal(u.atoms.elements, element_list)
+
+
+PDB_missing_ele = """\
+REMARK For testing warnings in case of absent element information
+ATOM      1  N   ASN A   1      -8.901   4.127  -0.555  1.00  0.00
+ATOM      2  CA  ASN A   1      -8.608   3.135  -1.618  1.00  0.00
+ATOM      3  C   ASN A   1      -7.117   2.964  -1.897  1.00  0.00
+ATOM      4  O   ASN A   1      -6.634   1.849  -1.758  1.00  0.00
+TER       5
+HETATM    6 CU    CU A   2      03.000  00.000  00.000  1.00 00.00
+HETATM    7 FE    FE A   3      00.000  03.000  00.000  1.00 00.00
+HETATM    8 Mg    Mg A   4      03.000  03.000  03.000  1.00 00.00
+TER       9
+"""
+
+def test_missing_elements_warnings():
+    """The test checks whether it asserts the appropriate warning on
```

Comment:
Since `assert` means something in python, I would instead suggest replacing that word with "gives".